core/sqf/conf/log4cxx.trafodion.masterexe.config

```diff
@@ -44,15 +45,15 @@ log4j.appender.mxoAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.mxoAppender.layout.ConversionPattern=%d, %p, %c, %m%n
 
 #MXOSRVR
+log4j.additive.mxoAppender=false 
 log4j.logger.MXOSRVR=ERROR, mxoAppender
 
 # SQL
 # uncomment the following line if you want to enable debug logging
-#log4j.logger.SQL.EXE=DEBUG
 log4j.logger.SQL=INFO
 log4j.logger.SQL.COMP=ERROR
-log4j.logger.SQL.EXE.Memory=INFO
-log4j.logger.SQL.ESP=ERROR
+log4j.logger.SQL.EXE=INFO
+log4j.logger.SQL.ESP=INFO
```

Comment:
The hiearchy of the logging level is TRACE, DEBUG, INFO,WARN,ERROR and FATAL,  It is good to have the logging level set to INFO to ensure that the users can decipher the problem without the need to prompte the logging level to the higher hierarchy. When a message is logged at the incorrect level, it can be corrected later